### PR TITLE
Fix index in config.py

### PR DESCRIPTION
**Pull request by Rantelo**
Also add guide comments

### CODE REVIEW

**Review by IleanaAldama — APPROVED** [review]

File: coati/generator.py [hunk 1]
```diff
@@ -73,32 +73,41 @@ def generate(project_name, ppt_path):
     spaces = " " * 12
     slide_tuples = '['
 
-    path_builders = os.path.join(path, 'builders/')
+    #Generate builders/ folder prior slides creation
+    _generate_path(os.path.join(path, 'builders/'))
+    log.warn('create folder %s', "./builders/")
```
Comment (lgovea) [inline coati/generator.py]: Are you sure this should be a warning? It looks more like an info log

Comment (Rantelo) [reply coati/generator.py]: The thing with info/warn is that we only use them to have different colors
warn is yellowish, and info is green. I wanted that folders use a different color :/

Comment (lgovea) [reply coati/generator.py]: I think the log message should be based on the thing that is doing rather than the visual representation

Comment (Rantelo) [reply coati/generator.py]: Yeah. Makes sense
I submitted the changes

**Pull request merged**